elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -33,6 +33,14 @@ pfe-navigation {
     }
   }
 
+  // Hide fallback login link if component has mounted
+  [pfelement],
```

Comment:
@kylebuch8 We mentioned deprecating the class `.PFElement` at one point in favor of `pfelement` attribute - is that still desired?  I ask because maybe only 1 of those selectors is needed here and the `.PFElement` option can be removed.  Both are added so you're safe either way.

Reply:
We can drop the .PFElement class here. The `.PFElement` class will be dropped in favor of the `pfelement` attribute that's added after upgrade.